### PR TITLE
Collection of configuration block fixes

### PR DESCRIPTION
**Pull request by frenck**
**Description:**

Collection of fixes to configuration blocks, in preparation for adding configuration validation to our build.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9874"></a>

### CODE REVIEW

File: source/_components/blackbird.markdown [hunk 1]
```diff
@@ -39,17 +39,19 @@ host:
 zones:
   description: This is the list of zones available. Valid zones are 1, 2, 3, 4, 5, 6, 7, 8. Each zone must have a name assigned to it.
   required: true
-  type: integer
+  type: map
   keys:
-    name:
+    ZONE_NUMBER:
```
Comment (klaasnicolaas) [inline source/_components/blackbird.markdown:44]: should this be in capital letters?

File: source/_components/blackbird.markdown [hunk 2]
```diff
@@ -39,17 +39,19 @@ host:
...
       description: The name of the zone.
+      type: string
 sources:
   description: The list of sources available. Valid source numbers are 1, 2, 3, 4, 5, 6, 7, 8. Each source number corresponds to the input number on the Blackbird matrix switch. Similar to zones, each source must have a name assigned to it.
   required: true
-  type: integer
+  type: map
   keys:
-    name:
+    ZONE_NUMBER:
```
Comment (klaasnicolaas) [inline source/_components/blackbird.markdown:52]: same

File: source/_components/blackbird.markdown (hunk 1 above)
Comment (frenck) [reply source/_components/blackbird.markdown:44]: It should be replaced by a zone number... so yes?

Comment (klaasnicolaas) [reply source/_components/blackbird.markdown:44]:
```suggestion
    zone_number:
```

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**